.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python: 2.7
+
+sudo: false
+
+# Pre-install packages for the ubuntu distribution
+cache:
+    apt: true
+    directories:
+        - $HOME/.cache/pip
+
+addons:
+    apt: true
+    packages:
+        - libatlas3gf-base
+        - libatlas-dev
+        - python-scipy
+        - cython
+
+# command to install dependencies
+install:
+    - pip install -U pip wheel
+    - pip install -r requirements.txt
+    - python setup.py build_ext --inplace
```

Comment:
I don't know if you need `build_ext` or `--inplace`.  I don't need those params when i run locally.